src/senaite/core/upgrade/v02_06_000.py

```diff
@@ -1064,6 +1064,65 @@ def migrate_manufacturers_to_dx(tool):
     logger.info("Convert Manufacturers to Dexterity [DONE]")
 
 
+@upgradestep(product, version)
+def migrate_storagelocations_to_dx(tool):
+    """Converts existing storage locations to Dexterity
+    """
+    logger.info("Convert StorageLocations to Dexterity ...")
+
+    # ensure old AT types are flushed first
+    remove_at_portal_types(tool)
```

Comment:
Remember to add `StorageLocation` and `StorageLocations` types in `REMOVE_AT_TYPES`. Otherwise, the following traceback arises when running the upgrade step:

```
Traceback (innermost last):
  Module ZPublisher.WSGIPublisher, line 176, in transaction_pubevents
  Module ZPublisher.WSGIPublisher, line 385, in publish_module
  Module ZPublisher.WSGIPublisher, line 288, in publish
  Module ZPublisher.mapply, line 85, in mapply
  Module ZPublisher.WSGIPublisher, line 63, in call_object
  Module Products.CMFPlone.controlpanel.browser.quickinstaller, line 666, in __call__
  Module Products.CMFPlone.controlpanel.browser.quickinstaller, line 399, in upgrade_product
  Module Products.GenericSetup.tool, line 1202, in upgradeProfile
  Module Products.GenericSetup.upgrade, line 185, in doStep
  Module senaite.core.upgrade, line 39, in wrap_func_args
  Module senaite.core.upgrade.v02_06_000, line 1078, in migrate_storagelocations_to_dx
  Module Products.GenericSetup.tool, line 375, in runImportStepFromProfile
  Module Products.GenericSetup.tool, line 1323, in _doRunImportStep
   - __traceback_info__: typeinfo
  Module Products.CMFCore.exportimport.typeinfo, line 222, in importTypesTool
  Module Products.GenericSetup.utils, line 935, in importObjects
   - __traceback_info__: portal_types
  Module Products.GenericSetup.utils, line 931, in importObjects
   - __traceback_info__: types/StorageLocation
  Module Products.GenericSetup.utils, line 530, in _importBody
  Module Products.CMFCore.exportimport.typeinfo, line 61, in _importNode
  Module Products.GenericSetup.utils, line 764, in _initProperties
ValueError: undefined property 'add_permission'
```

Reply:
`StorageLocation` and `StorageLocations` content types was be deleted after merge from 2.x branch :(

returned